blocks/melody/ff_container/ff_container-expandable/ff_container-expandable.less

```diff
@@ -1,29 +1,36 @@
-.ff_container-expandable__dropdown {
-	padding: @ff_size_spacing_med;
-	border: 1px solid @ff_colour_melody_int_blue;
-	color: @ff_colour_melody_int_blue;
-	cursor: pointer;
-	background: white;
-}
+.ff_container-expandable {
+	&__header {
+		padding: @ff_size_spacing_med;
+		border: 1px solid @ff_colour_melody_int_blue;
+		color: @ff_colour_melody_int_blue;
+		cursor: pointer;
+		background: white;
+	}
 
-.ff_container-expandable__dropdown--small-screens {
-	@media all and (min-width: @ff_bp_medium) { 
-		display: none;
+	&__icon {
+		width: 1.5em;
+		height: 1.5em;
+		float: right;
 	}
-}
 
-.ff_container-expandable__icon  {
-	width: 1.5em;
-	height: 1.5em;
-	float: right;
-}
+	&__content {
+		max-height: 1000px;
+		overflow: hidden;
+		transition: max-height 0.25s ease-in-out;
+	}
 
-.ff_container-expandable__content {
-	max-height: 1000px;
-	overflow: hidden;
-	transition: max-height 0.25s ease-in-out;
-}
+	&--is-collapsed &__content {
+		max-height: 0;
+	}
 
-.ff_container-expandable__content--hidden {
-	max-height: 0;
-}
+	&--is-widescreen &__header {
+		@media all and (min-width: @ff_bp_wide) { 
+			display: none;
+		}
+	}
+	&--is-widescreen &__content {
+		@media all and (min-width: @ff_bp_wide) { 
+			max-height: 1000px;
```

Comment:
When It's `--is-widescreen` (which means content should show and heading should disappear on widescreen) the javascript will apply the class of `is-collapsed` regardless of it.  `is-collapsed` sets the max-height: 0 to hide the content. So I want to set the max-height of the content to make it visible on widescreens. This max-height thing is only for animation purposes.